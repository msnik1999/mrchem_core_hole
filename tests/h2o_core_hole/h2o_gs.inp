# vim:syntax=sh:

world_prec = 1.0e-3                 # Overall relative precision
world_size = 6                      # Size of simulation box 2^n

MPI {
  numerically_exact = true          # Guarantee identical results in MPI
}

Molecule {
$coords
O       0.0000  0.0000  -0.1250
H      -1.4375  0.0000   1.0250
H       1.4375  0.0000   1.0250
$end
}

WaveFunction {
  method = PBE                      # Wave function method (HF or DFT)
}

SCF {
  run = false
  localize = true                   # Use canonical of localized orbitals
  write_orbitals = true
  path_orbitals = initial_guess
}