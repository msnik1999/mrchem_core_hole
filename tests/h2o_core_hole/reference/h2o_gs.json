{
  "input": {
    "constants": {
      "N_a": 6.02214076e+23,
      "angstrom2bohrs": 1.8897261246257702,
      "boltzmann_constant": 1.380649e-23,
      "dipmom_au2debye": 2.5417464739297717,
      "e0": 8.8541878128e-12,
      "electron_g_factor": -2.00231930436256,
      "elementary_charge": 1.602176634e-19,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084,
      "meter2bohr": 18897261246.2577
    },
    "geom_opt": {
      "init_step_size": -0.5,
      "max_force_component": 0.005,
      "max_history_length": 10,
      "max_iter": 100,
      "minimal_step_size": 0.01,
      "run": false,
      "subspace_tolerance": 0.001,
      "use_previous_guess": false
    },
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "o",
          "r_rms": 5.0580178957e-05,
          "xyz": [
            0.0,
            0.0,
            -0.125
          ]
        },
        {
          "atom": "h",
          "r_rms": 2.6569547399e-05,
          "xyz": [
            -1.4375,
            0.0,
            1.025
          ]
        },
        {
          "atom": "h",
          "r_rms": 2.6569547399e-05,
          "xyz": [
            1.4375,
            0.0,
            1.025
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "omp_threads": -1,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 5,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "gs",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "nuclear_model": "point_like",
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "environment": "None",
        "external_field": "None",
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE)",
        "prec": 0.001,
        "relativity": "None",
        "restricted": true,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "occupancies": [],
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "write_orbitals": {
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        4.6220056999883845e-18,
        0.0,
        0.003683095683220021
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 1.3104652688004346,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            8.076276248144426e-05,
            0.0,
            1.3104652663117673
          ],
          "vector_el": [
            8.076276226850348e-05,
            0.0,
            0.2604652663122011
          ],
          "vector_nuc": [
            0.0,
            0.0,
            1.0499999999995662
          ]
        }
      },
      "geometry": [
        {
          "symbol": "O",
          "xyz": [
            0.0,
            0.0,
            -0.125
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            -1.4375,
            0.0,
            1.025
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            1.4375,
            0.0,
            1.025
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -18.231417411078613,
          -0.7600557792145582,
          -0.3227111066501499,
          -0.09752778987848379,
          -0.024339907214931882
        ],
        "occupation": [
          2.0,
          2.0,
          2.0,
          2.0,
          2.0
        ],
        "spin": [
          "p",
          "p",
          "p",
          "p",
          "p"
        ],
        "sum_occupied": -38.87210398807347
      },
      "orbital_positions": [
        [
          -4.640132875508272e-08,
          -1.0573500855943139e-15,
          -0.12426224618405877
        ],
        [
          -0.7134623907846172,
          -2.6066223814682346e-14,
          0.4386510234174848
        ],
        [
          0.7134025102517081,
          3.269090784895313e-14,
          0.4386424799209419
        ],
        [
          9.772776574859159e-06,
          -0.4887792723168228,
          -0.44163194515522336
        ],
        [
          9.77277652859938e-06,
          0.48877927231682455,
          -0.4416319451552452
        ]
      ],
      "scf_energy": {
        "E_ee": 48.699940065644626,
        "E_eext": 0.0,
        "E_el": -84.85121005186711,
        "E_en": -200.924676817657,
        "E_kin": 76.83379945539507,
        "E_next": 0.0,
        "E_nn": 9.039235444235468,
        "E_nuc": 9.039235444235468,
        "E_tot": -75.81197460763164,
        "E_x": 0.0,
        "E_xc": -9.460272755249804,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 1,
      "routine": "mrchem.x",
      "total_cores": 1,
      "version": "1.2.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 48.699940065644626,
        "E_eext": 0.0,
        "E_el": -84.85121005186711,
        "E_en": -200.924676817657,
        "E_kin": 76.83379945539507,
        "E_next": 0.0,
        "E_nn": 9.039235444235468,
        "E_nuc": 9.039235444235468,
        "E_tot": -75.81197460763164,
        "E_x": 0.0,
        "E_xc": -9.460272755249804,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}