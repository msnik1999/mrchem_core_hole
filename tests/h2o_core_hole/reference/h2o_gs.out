***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         mom_merged                               ***
***         Git commit hash    ef9fe2457e9ead3325c3-dirty               ***
***         Git commit author  Niklas GÃ¶llmann                         ***
***         Git commit date    Fri May 10 10:46:19 2024 +0200           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               1
 Total cores             :                                               1

---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.6.0-alpha
 Git branch            : mom
 Git commit hash       : 9ebc1da6290a65a252e6
 Git commit author     : Laura Ratcliff
 Git commit date       : Wed Jun 14 14:11:39 2023 +0200

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (1 threads)

---------------------------------------------------------------------------



===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================



***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************


===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       O            :        0.000000        0.000000       -0.125000
    1       H            :       -1.437500        0.000000        1.025000
    2       H            :        1.437500        0.000000        1.025000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.003683
===========================================================================



***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-03
 Screening               : 1.20000e+01 StdDev
 Restricted              : True
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                           5.000
 Beta electrons          :                                           5.000
 Total electrons         :                                          10.000
---------------------------------------------------------------------------
    n     Occ Spin       :                                            Norm
---------------------------------------------------------------------------
    0    2.00    p       :                              9.999999999719e-01
    1    2.00    p       :                              9.999999993851e-01
    2    2.00    p       :                              9.999999997831e-01
    3    2.00    p       :                              9.999999993963e-01
    4    2.00    p       :                              9.999999999731e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE)
 Relativity              : None
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                 76.833799455395
 E-N energy              :            (au)               -200.924676817657
 Coulomb energy          :            (au)                 48.699940065645
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -9.460272755250
 N-N energy              :            (au)                  9.039235444235
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -84.851210051867
 Nuclear energy          :            (au)                  9.039235444235
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.581197460763e+01
                         :      (kcal/mol)             -4.757273231372e+04
                         :        (kJ/mol)             -1.990443120006e+05
                         :            (eV)             -2.062948923119e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n     Occ Spin       :                                         Epsilon
---------------------------------------------------------------------------
    0    2.00    p       :            (au)                -18.231417411079
    1    2.00    p       :            (au)                 -0.760055779215
    2    2.00    p       :            (au)                 -0.322711106650
    3    2.00    p       :            (au)                 -0.097527789878
    4    2.00    p       :            (au)                 -0.024339907215
---------------------------------------------------------------------------
 Sum occupied            :            (au)                -38.872103988073
===========================================================================



***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************


===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       O            :        0.000000        0.000000       -0.125000
    1       H            :       -1.437500        0.000000        1.025000
    2       H            :        1.437500        0.000000        1.025000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.003683
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                 76.833799455395
 E-N energy              :            (au)               -200.924676817657
 Coulomb energy          :            (au)                 48.699940065645
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -9.460272755250
 N-N energy              :            (au)                  9.039235444235
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -84.851210051867
 Nuclear energy          :            (au)                  9.039235444235
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.581197460763e+01
                         :      (kcal/mol)             -4.757273231372e+04
                         :        (kJ/mol)             -1.990443120006e+05
                         :            (eV)             -2.062948923119e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n     Occ Spin       :                                         Epsilon
---------------------------------------------------------------------------
    0    2.00    p       :            (au)                -18.231417411079
    1    2.00    p       :            (au)                 -0.760055779215
    2    2.00    p       :            (au)                 -0.322711106650
    3    2.00    p       :            (au)                 -0.097527789878
    4    2.00    p       :            (au)                 -0.024339907215
---------------------------------------------------------------------------
 Sum occupied            :            (au)                -38.872103988073
===========================================================================


===========================================================================
                             Orbital Positions
---------------------------------------------------------------------------
Index                    :               x               y               z
---------------------------------------------------------------------------
    0                    :       -0.000000       -0.000000       -0.124262
    1                    :       -0.713462       -0.000000        0.438651
    2                    :        0.713403        0.000000        0.438642
    3                    :        0.000010       -0.488779       -0.441632
    4                    :        0.000010        0.488779       -0.441632
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000081       -0.000000        0.260465
 Magnitude               :            (au)                        0.260465
                         :         (Debye)                        0.662037
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        1.050000
 Magnitude               :            (au)                        1.050000
                         :         (Debye)                        2.668834
---------------------------------------------------------------------------
 Total vector            :        0.000081        0.000000        1.310465
 Magnitude               :            (au)                        1.310465
                         :         (Debye)                        3.330870
===========================================================================




***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m 47s                       ***
***                                                                     ***
***************************************************************************